src/routes/mod.rs

```diff
@@ -68,6 +70,7 @@ impl actix_web::ResponseError for ApiError {
             ApiError::DatabaseError(..) => actix_web::http::StatusCode::INTERNAL_SERVER_ERROR,
             ApiError::AuthenticationError => actix_web::http::StatusCode::UNAUTHORIZED,
             ApiError::JsonError(..) => actix_web::http::StatusCode::BAD_REQUEST,
+            ApiError::SearchError(..) => actix_web::http::StatusCode::FAILED_DEPENDENCY,
```

Comment:
This error code doesn't appear to be accurate for this error.  `INTERAL_SERVER_ERROR` seems more correct, or something like 503 `SERVICE_UNAVAILABLE` if meilisearch is down.